doc/aaa/TACACS+ Design.md

```diff
@@ -0,0 +1,537 @@
+
+## 4.2 Accounting Implementation
+ - [Auditd](#auditd) will enable on SONiC to provide syscall event for accounting.
+ - [audisp-tacplus](#audisp-tacplus) is a Auditd plugin that support TACACS+ Accounting (user command).
+ - Pam_tacplus will provide session accounting, please check [TACACS+ Authentication](#TACPLUS-Authentication)
+
+The following figure show how audisp-tacplus work with TACACS+ server.
+```
+         +-----------------+
+         |  Syscall Event  |
+         +-------+---------+
+                 |
++----------------v---------------+       +---------------------+
+| Auditd                         |       |                     |
+|                                |       |                     |
+|   +-------------------------+  |       |                     |
+|   |     audisp-tacplus      +---------->    TACACS+ server   |
+|   +-------------------------+  |       |                     |
+|                                |       |                     |
+|                                |       |                     |
++---------------+----------------+       +---------------------+
+```
+
+
+ - The hostcfg enforcer reads data from configDB to configure host environment.
+   - The AAA config module in hostcfg enforcer is responsible for modifying Auditd configuration files in host. this will be low priority.
+   - For how TACACS+ config file update, please check [TACACS+ Authentication](#TACPLUS-Authentication)
+
+The following figure show how Auditd config an TACACS+ config update by ConfigDB.
+```
++--------------------------------+       +---------------------+
+| Auditd                         |       |                     |
+|   +-------------------------+  |       |  +--------------+   |
+|   | Auditd config file      <-------------+ Accounting   |   |
+|   |                         |  |       |  |    Config    |   |
+|   +-------------------------+  |       |  +--------------+   |
+|                                |       |                     |
+|   +-------------------------+  |       |  +--------------+   |
+|   | TACACS+ config file     <-------------+  AAA Config  |   |
+|   +-------------------------+  |       |  +--------------+   |
+|                                |       |                     |
+|                                |       |  HostCfg Enforcer   |
++--------------------------------+       +----------^----------+
+                                                    |
+           +---------+                      +-------+--------+
+           |         |                      |                |
+           |   CLI   +---------------------->    ConifgDB    |
+           |         |                      |                |
+           +---------+                      +----------------+
+```
+
+
+## 4.3 ConfigDB Schema
+ - Existing tables, for more detail please check [TACACS+ Authentication](#TACPLUS-Authentication)
```

Comment:
>[TACACS+ Authentication](#TACPLUS-Authentication) [](http://example.com/codeflow?start=49&length=49)

Is it a wrong link?
I guess [https://github.com/Azure/SONiC/blob/master/doc/aaa/TACACS%2B%20Authentication.md#aaa-table-schema](https://github.com/Azure/SONiC/blob/master/doc/aaa/TACACS%2B%20Authentication.md#aaa-table-schema)

Reply:
Fixed.